src/CallingExtensions.js

```diff
@@ -46,6 +46,18 @@ class CallingExtensions {
     });
   }
 
+  userAvailable() {
+    this.sendMessage({
+      type: messageType.USER_AVAILABLE,
```

Comment:
Is the user unavailable until we receive this event?

Reply:
Should we add `userAvailable` and `userUnavailable` to demo minimal js?

Reply:
Yup! The calling provider will also have the option to let us know of if the user is available upon initialization:
```js
const payload
{
   // Whether a user is available for inbound calls
  isUserAvailable: true|false,
  // Whether a user is logged-in
  isLoggedIn: true|false,
  // Optionally send the desired widget size
  sizeInfo: {
    height: number,
    width: number
  }
}

extensions.initialized(payload);
```

Reply:
Yes!